projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/case-flag/components/case-flag-summary-list/case-flag-summary-list.component.ts

```diff
@@ -1,21 +1,22 @@
-import { Component, Input, OnInit } from '@angular/core';
+import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
 import { FlagDetailDisplay } from '../../domain';
-import { CaseFlagSummaryListDisplayMode } from '../../enums';
+import { CaseFlagFieldState, CaseFlagSummaryListDisplayMode } from '../../enums';
 
 @Component({
   selector: 'ccd-case-flag-summary-list',
   templateUrl: './case-flag-summary-list.component.html'
 })
 export class CaseFlagSummaryListComponent implements OnInit {
-
   @Input() public flagForSummaryDisplay: FlagDetailDisplay;
   @Input() public summaryListDisplayMode: CaseFlagSummaryListDisplayMode;
+  @Output() public changeButtonEmitter = new EventEmitter<number>();
 
   public flagDescription: string;
   public flagComments: string;
   public flagStatus: string;
   public displayMode = CaseFlagSummaryListDisplayMode;
   public addUpdateFlagHeaderText: string;
+  public CaseFlagFieldState = CaseFlagFieldState;
```

Comment:
@MateiHMCTS - really minor one but can this be changed to camelCase instead of PascalCase (for consistency)?